Purchases.xcodeproj/project.pbxproj

```diff
@@ -2372,6 +2357,63 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		B3D5CFBB267282630056FA67 /* XCRemoteSwiftPackageReference "Nimble" */ = {
```

Comment:
That's my understanding, it's a dependency to build the test targets. Xcode *might* download them automatically because it could assume you want to make sure the tests pass, but I'm not sure.

Reply:
Just tested with a new project, I pointed SPM to this branch and it only checked out the SDK 🎉

<img width="1106" alt="Screen Shot 2021-06-11 at 2 32 37 PM" src="https://user-images.githubusercontent.com/1304821/121750780-e25a4e00-cac1-11eb-9364-b09483cf7820.png">

Reply:
:chefskiss: